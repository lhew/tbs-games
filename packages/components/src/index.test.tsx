import React from 'react';
import TestRenderer from 'react-test-renderer'; // ES6
import { Button, FormField, Popup, Sidebar, Toaster } from '.';
test('Testing imports', () => {
  describe('Mounting a simple page with the components', () => {
    const defaultInput = TestRenderer.create(
      <div>
        <Sidebar>
          <h1>Test</h1>
        </Sidebar>
        <Button>a button</Button>
        <Popup>A popup example</Popup>
        <FormField name="foo" type="text" />
        <Toaster>Toaster</Toaster>
      </div>,
    );
    expect(defaultInput).toMatchSnapshot();
  });
});
