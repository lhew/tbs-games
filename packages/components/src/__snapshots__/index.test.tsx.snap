// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing imports 1`] = `
<div>
  <div
    className="sc-AxhCb bRrSKR"
  >
    <h1>
      Test
    </h1>
  </div>
  <button
    className="sc-AxjAm kwfnmA"
  >
    a button
  </button>
  <div
    className="sc-AxiKw kWKImv"
  >
    <div
      className="sc-AxirZ gHoGsv"
    >
      A popup example
    </div>
  </div>
  <div
    className="sc-AxheI cywBwt"
  >
    <input
      className="sc-fzozJi fbOHBz"
      name="foo"
    />
  </div>
  <div
    className="sc-AxgMl depUzd"
  >
    Toaster
  </div>
</div>
`;
